src/lib/default.nix

```diff
@@ -0,0 +1,40 @@
+# Licensed under GNU Lesser General Public License v3 or later, see COPYING.
+# Copyright (c) 2019 Alexander Sosedkin and other contributors, see AUTHORS.
+
+{ callPackage }:
+
+let
+  ids = import (callPackage ./ids.nix { });
+
+  userName = "nix-on-droid";
+in
+
+{
+  buildConfig = { arch, initialBuild, nixOnDroidChannelURL }: (
+    {
+      channel = {
+        nix-on-droid = nixOnDroidChannelURL;
+        nixpkgs = "https://nixos.org/channels/nixos-19.09";
```

Comment:
I'm confused. Why does the default channel URL live here now? It only takes place on initial execution (and after wipes), so I think it is sensible to configure it at zipball build time, but not during normal usage.